lib/Net/Async/Github.pm

```diff
@@ -1311,7 +1349,7 @@ Will raise an exception on invalid input.
 sub validate_repo_name {
     my ($self, $repo) = @_;
     die "repo name not defined" unless defined $repo;
-    die "repo name contains invalid characters" if $repo =~ /[^a-z0-9-]/i;
+    die "repo name contains invalid characters" if $repo =~ /[^a-z0-9-_]/i;
```

Comment:
A repo name wouldn't necessarily be "invalid", but by convention, the usage of `_` is discouraged
https://github.com/bcgov/BC-Policy-Framework-For-GitHub/blob/master/BC-Gov-Org-HowTo/Naming-Repos.md

Reply:
The problem is that our existing repos have such repo names.

Reply:
okay